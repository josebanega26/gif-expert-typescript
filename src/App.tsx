import React, { useState } from 'react';
import Footer from './Components/Footer';
import AddCategory from './Components/AddCategory';
import CategoriesList from './Components/CategoriesList';

function App() {
  const [categories, setCategories] = useState(['Star wars', 'Lord of rings']);

  return (
    <>
      <div className='container'>
        <h2>Gif Expert App</h2>
        <AddCategory setCategories={setCategories}></AddCategory>
        <CategoriesList data={categories} />
      </div>
      <Footer></Footer>
    </>
  );
}

export default App;
