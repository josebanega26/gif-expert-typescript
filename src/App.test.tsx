import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App tests', () => {
  let wrapper: any;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('Component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
