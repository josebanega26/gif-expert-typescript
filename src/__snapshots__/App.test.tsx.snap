// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests Component should render 1`] = `
<Fragment>
  <div
    className="container"
  >
    <h2>
      Gif-Expert with typescript
    </h2>
    <AddCategory
      setCategories={[Function]}
    />
  </div>
  <Footer />
</Fragment>
`;
